tests/integration_tests/allennlp_tests/example_with_executor_and_pruner.jsonnet

```diff
@@ -60,6 +60,7 @@ local DROPOUT = std.parseJson(std.extVar('DROPOUT'));
     callbacks: [
       {
         type: 'optuna_pruner',
+        monitor: 'best_validation_loss',
```

Comment:
After this PR is merged, `monitor` is required to specify in a config file.
I will document it.

Reply:
Is it documented anywhere?

Reply:
I found we can use a metric specified in `AllenNLPExecutor` in e1da97c.
This keeps compatibility and removed this line in 261b4ea.